pages/styles/index.css

```diff
@@ -83,10 +83,12 @@ div[id^="inventory-"] aside,
 }
 
 #inventory-top > div > div {
-  display: inline-block;
-  margin: 8px 16px;
-  height: 24px;
-  vertical-align: middle;
+  /* display: inline-block; */
+  /* margin: 8px 16px; */
+  padding: 3px 8px 0px 8px;
+  /* margin-top: 3px; */
+  /* height: 24px; */
+  /* vertical-align: middle; */
```

Comment:
I'm not sure whats going on here looks like some spacing stuff. I suppose you could delete if it looks the same? I see the inventory is shorter compared to production.